airbyte-integrations/connectors/source-google-directory/source_google_directory/spec.json

```diff
@@ -78,13 +78,5 @@
         ]
       }
     }
-  },
-  "authSpecification": {
```

Comment:
do we automatically convert authSpecification to advancedAuth or is this a breaking change?

Reply:
@girarda this is not a breaking change. Both advancedAuth and authSpecification are nullable, besides this connector is outdated on cloud and has no oauth support so we will not break things